research/src/pages/radio-button/radio-button.proposal.mdx

```diff
@@ -0,0 +1,185 @@
+---
+menu: Proposals
+name: Radio Button (Editor's Draft)
```

Comment:
I didn't change anything at first because the rest of the discussion did not advance. But as soon as I get a clear requirement from the reviewers, I will change it.